### ISSUE TITLE
[Feature Request] Improved Code Block

### ISSUE THREAD

**Issue by dom6770** [notea-org/notea#137]
Notea is my go-to note app and I love it. One of my use-cases for the app are my installation steps for servers, etc. So I have a lot of commands to install and configure a plain Linux server. Unfortunately, with Notea it's just a bit annoying. My issues are

a) There's no button to easily copy code. 
b) Inside the code block I can't press CTRL + A to select everything, because it won't select just the code block, it will select the whole note section.
c) While hovering on a code block, which has a long code inside, the language selector will obscure the view
![image](https://user-images.githubusercontent.com/11932291/155306486-e9b293f8-87a2-4381-93c5-64915ef51c3f.png)

My idea to improve the code block would be

a) moving the language selector above the code block and adding a copy button, something like [this](https://www.dannyguo.com/blog/how-to-add-copy-to-clipboard-buttons-to-code-blocks-in-hugo/). 
b) limiting the CTRL + A hotkey to the code block itself.

**Comment by ExXxtr3me2** [comment]
It´s also strange that there is a copy button when you share the note and open it through the shared link.

**Comment by tecc** [comment]
![image](https://user-images.githubusercontent.com/38840234/196567323-89e17bf2-0516-4504-8861-b2c75e19e3d7.png)
Is this better? I mocked it up whilst making the new editor.

**Comment by ExXxtr3me** [comment]
Much better. 👍
Will there be a new updated docker in the near future?

**Comment by tecc** [comment]
Yes, there definitely will be, once I've managed to stabilise the image.
There are also automatically published images that are bleeding-edge (the tag name is `main`, if you're interested) - last I checked, the image wasn't even functional.

I would like to point out that the CTRL-A shortcut does *not* work within the code block, which is mildly annoying. I tried to fix it, but apparently the editor code doesn't like me.
